packages/auth-ts/src/core/strategies/auth_credential.ts

```diff
@@ -122,7 +125,7 @@ export function authCredentialFromTokenResponse(
         pendingToken,
         idToken: oauthIdToken,
         accessToken,
-        signInMethod: providerId as SignInMethod
+        signInMethod: (providerId as unknown) as SignInMethod
```

Comment:
This is funky

Reply:
It wont compile without it: "There is no overlap between these two types"

Reply:
Try casting to string instead of unknown then